recipes/timml/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "timml" %}
+{% set version = "5.0.1" %}
+{% set sha256 = "2790663285e22e5a58dfe38f16166c0b09b610890205d98996fd0c72491cbb18" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
+  url: https://github.com/mbakker7/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install 
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+  host:
+    - m2w64-toolchain  # [win]
+    - libpython  # [win]
+    - python 
+    - numpy >=1.12 
+    - scipy
+    - matplotlib >=2.0
+  run:
+   - python 
+   - {{ pin_compatible('numpy') }} 
+   - scipy
+   - matplotlib >=2.0
+
+test:
+  imports:
+    - timml
+
+about:
+  home: https://github.com/mbakker7/timml
+  license: MIT
+  license_file: LICENSE
+  summary: 'TimML is a multi-layer analytic element model'
+  dev_url: https://github.com/mbakker7/timml
+  
+extra:
+  recipe-maintainers:
+    - mbakker7
```

Comment:
@mbakker7 - are you OK with being a maintainer here?

Reply:
Yes, absolutely. I am very glad @jentjr submitted this and I will happily be the maintainer.